### PR TITLE
chore: migrated admin router to drizzle and changed createdAt schema …

### PR DESCRIPTION
**Pull request by JohnAllenTech**
…mode to date

# ✨ Codu Pull Request 💻

Codu Logo

## Pull Request details

- Migrated admin router to drizzle
- Tested banning and unbanning users. working 100%

## Any Breaking changes

- None

## Associated Screenshots

- None

## Summary by CodeRabbit

- **Refactor**
	- Improved database interactions for user management operations to enhance performance.
- **New Features**
	- Updated data handling for better accuracy and efficiency in processing dates across various entities.

### CODE REVIEW

File: server/api/router/admin.ts [hunk 1]
```diff
@@ -10,43 +12,36 @@ export const adminRouter = createTRPCRouter({
       const { userId, note } = input;
       const currentUserId = ctx.session.user.id;
 
-      const user = await ctx.prisma.user.findFirstOrThrow({
-        where: {
-          id: userId,
-        },
+      const user = await ctx.db.query.user.findFirst({
+        where: (user, { eq }) => eq(user.id, userId),
```
[the remark this answers is not in the document]
Comment (JohnAllenTech) [reply server/api/router/admin.ts:16]: what is drizzles equivalent method for finding a user ?

[the remark this answers is not in the document]
Comment (JohnAllenTech) [reply server/api/router/admin.ts:16]: No it doesnt... findOne isnt even a function in drizzle. r/confidentlyincorrect would be a great place for you to look up

[the remark this answers is not in the document]
Comment (John-Paul-Larkin) [reply server/api/router/admin.ts:16]: > No it doesnt... findOne isnt even a function in drizzle. r/confidentlyincorrect would be a great place for you to look up

Haha. I was just thinking I don't remember seeing a findOne, and I was looking for it yesterday.

[the remark this answers is not in the document]
Comment (JohnAllenTech) [reply server/api/router/admin.ts:16]: yeah it parsed the drizzle docs and found this https://orm.drizzle.team/learn/migrate/migrate-from-typeorm#replace-select-queries and put 1 and 1 together and made 5

Stuff like this really puts into perspective how much AI lacks context

[the remark this answers is not in the document]
Comment (John-Paul-Larkin) [reply server/api/router/admin.ts:16]: Good catch. That makes sense.
I imagine docs will evolve to some kind of standard which makes them clearer for ai parsing.
Either way it is going to improve, but mistakes like this ease the existential dread 🙈

**Review by NiallJoeMaher — APPROVED** [review]
🌮 LGTM!

**Pull request merged**